content/en/docs/Installation/azure.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Installation on Microsoft Azure"
+linkTitle: "Installation on Azure"
+weight: 3
+date: 2025-03-24
+description: Install Kairos on Microsoft Azure
+---
+
+This page describes how to install Kairos on [Microsoft Azure](https://azure.microsoft.com/) after you have created a disk image. Since release v3.3.2, Kairos pipeline is pushing a public OS image to Azure which you can use.
```

Comment:
do we want to mention 3.3.5 instead which we know its the first one working properly for cloud?

Reply:
yup, good point